x-pack/plugins/cloud_integrations/cloud_data_migration/public/application/components/app.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+
+import { CoreStart, CoreTheme } from '@kbn/core/public';
+
+import { css } from '@emotion/react';
+import { KibanaThemeProvider, KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { Observable } from 'rxjs';
+import { BreadcrumbService } from '../services/breadcrumbs';
+
+interface CloudDataMigrationAppDeps {
+  http: CoreStart['http'];
+  breadcrumbService: BreadcrumbService;
+  theme$: Observable<CoreTheme>;
+}
+
+export const CloudDataMigrationApp = ({
+  http,
+  breadcrumbService,
+  theme$,
+}: CloudDataMigrationAppDeps) => {
+  const basePath = http.basePath.get() ?? '';
+  const { euiTheme } = useEuiTheme();
+  const listItemProps = {
+    iconType: 'checkInCircleFilled',
+    iconProps: { color: 'success' },
+    wrapText: true,
+  };
+  const listItemCss = css`
+    font-weight: 300;
+  `;
+  const panelPaddingCss = css`
+    padding: calc(${euiTheme.size.xxxl});
+    margin: ${euiTheme.size.l} auto;
+    width: 100%;
+    max-width: 875px;
+  `;
+
+  useEffect(() => {
+    breadcrumbService.setBreadcrumbs('home');
+  }, [breadcrumbService]);
+
+  return (
+    <KibanaThemeProvider theme$={theme$}>
+      <KibanaContextProvider
+        services={{
+          breadcrumbService,
+        }}
+      >
+        <EuiPanel css={panelPaddingCss} color="subdued">
+          <EuiFlexGroup
+            direction="column"
+            alignItems="center"
+            gutterSize="xl"
+            justifyContent="spaceBetween"
+            css={css`
+              max-width: 500px;
+              margin: 0 auto;
+            `}
+          >
+            <EuiFlexItem>
+              <EuiImage
+                alt={i18n.translate('cloudMigration.illustration.alt.text', {
+                  defaultMessage: 'Illustration for cloud data migration',
+                })}
+                css={css`
+                  max-width: 75%;
```

Comment:
[nit] Can you create a const for this css to consistent with `listItemCss` and `panelPaddingCss` as defined above?